spec/factories/value_definition.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :v2_value_definition, class: V2::ValueDefinition do
+    id { SecureRandom.uuid }
```

Comment:
Again, we are not in a view, but a regular table and PG generates this for us. You can delete it.